.github/workflows/maven.yml

```diff
@@ -0,0 +1,102 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build_and_test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ 'ubuntu-latest', 'macos-latest', 'windows-latest' ]
+        java-version: [ 8 ]
+        browser: ['jbrowser', 'googlechromeheadless', 'firefoxheadless'] # 'htmlunitwithjs' not working with Selenium4
```

Comment:
> # 'htmlunitwithjs' not working with Selenium4

:-(

Reply:
Selenium4 is still Alpha, so it's not surprise that there's no htmlunit for Selenium 4 (yet).